docker/etna-apache/Dockerfile

```diff
@@ -9,7 +9,7 @@ ENV DOCKERIZE_URL https://github.com/jwilder/dockerize/releases/download/v${DOCK
 RUN groupadd -g 1002 runner
 RUN useradd -ms /bin/bash -u 1002 -g 1002 runner
 RUN apt-get update -y
-RUN apt-get install -y libapache2-mod-xsendfile libapache2-mod-shib2 xmlsec1 curl
+RUN apt-get install -y libapache2-mod-xsendfile libapache2-mod-shib xmlsec1 curl
```

Comment:
weirdly the package got renamed in the debian 11, which is what the upgraded image is based off.